configuration/raspi-config.md

```diff
@@ -15,27 +15,25 @@ sudo raspi-config
 
 The `sudo` is required because you will be changing files that you do not own as the `pi` user.
 
-You should see a blue screen with options in a grey box in the centre, like so:
+You should see a blue screen with options in a grey box in the centre, something like so (this might not be exactly what you see):
```

Comment:
Not sure about 'should'. What about:

`A blue menu screen will be displayed, with options in a grey box:`

<picture>

`Note that the menu shown may differ slightly.`

Other than that, I think AndrewS has spotted everything.